utils/cliutils/commandsflags.go

```diff
@@ -446,6 +449,9 @@ const (
 	// *** Completion Commands' flags ***
 	Completion = "completion"
 	Install    = "install"
+
+	// Setup flags
+	setupFormat = "setup-format"
```

Comment:
setupFormat = "setup-format"
-->
outputFormat = "output-format"

Reply:
I chose a specific name for the --format option of the setup command because it's hidden, which makes it different from the --format option in other commands.